src/tests/tablet/homeUI/CurrentMeetingValuesVerifications.java

```diff
@@ -0,0 +1,158 @@
+package tests.tablet.homeUI;
+
+import static tests.tablet.homeUI.HomeTabletSettings.createMeetingFromExcel;
+import static tests.tablet.homeUI.HomeTabletSettings.getAuthenticationValue;
+import static tests.tablet.homeUI.HomeTabletSettings.getMeetingSubject;
+import static tests.tablet.homeUI.HomeTabletSettings.getRoomDisplayName;
+
+import java.io.IOException;
+
+import org.apache.log4j.Logger;
+import org.apache.log4j.PropertyConfigurator;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SchedulePage;
+import framework.rest.RootRestMethods;
+
+/**
+ * This TCs verify that {now} tile is set whit correct information
+ * when a meeting created in the current time.
```

Comment:
might be the correct tittle is:

This TCs verify that {now} tile is set whit correct information
when a meeting is created in the current time.